Containers/borgbackup/Dockerfile

```
@@ -1,14 +1,13 @@
-FROM debian:bullseye-20221219-slim
+FROM alpine:3.17.0
 
 RUN set -ex; \
     \
-    echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list; \
-    apt-get update; \
-    apt-get install -y --no-install-recommends borgbackup -t bullseye-backports; \
-    apt-get install -y --no-install-recommends \
+    apk add --update --no-cache \
```

```suggestion
    apk add --no-cache \
```

Because of:
```
 apk add --help | grep update
  -U, --update-cache    Alias for '--cache-max-age 1'
```
Or does `--update` something, but the help doesn't show it?

---

> Or does `--update` something, but the help doesn't show it?

yes, it does :)

---

https://stackoverflow.com/a/59052608

---

But why do we say?:
 - Don't create or use a cache (--no-cache)
and:
- Create a cache (--update-cache)

---

ah, that is a good point. I guess will have to investigate...

---

https://github.com/nextcloud/all-in-one/pull/1767

---

done